src/popup/components/TransactionListItem.vue

```diff
@@ -108,10 +109,13 @@ export default defineComponent({
     let timerInterval: NodeJS.Timer;
     const transactionDate = ref();
 
-    const currentTransaction = computed(
-      () => (props.multisigTransaction || props.transaction),
+    const currentTransaction = computed<ITransaction>(
```

Comment:
I strongly suggest to type the callback return type instead of just passing the type as the generic argument.